.golangci.yml

```diff
@@ -1,10 +1,9 @@
 run:
   deadline: 10m
-
-linters:
   build-tags:
     - rpctest
 
+linters:
```

Comment:
It looks to me like this change will make it so the tests gated behind the `rpctest` build tag will not be linted.  I don't think that's what we want.